src/loader/BlendLoader.py

```diff
@@ -28,6 +49,17 @@ class BlendLoader(LoaderInterface):
 
     Result: loading all objects from folder /Object of file.blend that match the pattern.
 
+    Note:
+    Some datablocks types like bpy.types.Light, bpy.types.Mesh, bpy.types.Camera etc are designed to be wrapped in 
+    bpy.types.Object. Loading the container bpy.types.Object (represented as /Object datablock here after) also loads the
```

Comment:
> /Object

I think we should drop the slash now altogether and use it when resolving path only since now its more confusing. What do you think?